x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/inspect/inspect.spec.ts

```diff
@@ -45,7 +45,7 @@ describe('Inspect', () => {
       toggleTimelineVisibility();
       executeKQL(hostExistsQuery);
       cy.get(TIMELINE_SETTINGS_ICON).trigger('click', { force: true });
-      cy.get(TIMELINE_INSPECT_BUTTON).should('not.be.disabled', { timeout: DEFAULT_TIMEOUT });
```

Comment:
maybe instead we could adjust default timeout on the config level?
https://docs.cypress.io/guides/references/configuration.html#Timeouts

Reply:
Completely agree with you in this, I'll do that in the refactor I'm working on 😊